src/js/components/MemoryFieldComponent.jsx

```diff
@@ -0,0 +1,27 @@
+var React = require("react/addons");
+
+var MemoryFieldComponent = React.createClass({
+  displayName: "MemoryFieldComponent",
+  propTypes: {
+    megabytes: React.PropTypes.number.isRequired
+  },
+  render() {
+    // For a documentation of the different unit prefixes please refer to:
+    // https://en.wikipedia.org/wiki/Template:Quantities_of_bytes
+    const units = ["MB", "GB", "TiB", "PiB", "EiB", "ZiB", "YiB"];
```

Comment:
Eheh! We're probably covering the 0.000000000000000001% case but – heck, why not? :)